packages/slice-machine/server/src/api/slices/push.ts

```diff
@@ -21,7 +21,7 @@ export async function pushSlice(
 
   try {
     const smModel: SliceSM = IO.Slice.readSlice(modelPath);
-    const { err } = await purge(env, slices, sliceName);
+    const { err } = await purge(env, slices, smModel);
```

Comment:
same here "why are you passing the whole model down? Think it's better if you just pass down the slice ID if that's all you need"